src/_pytest/python.py

```diff
@@ -1503,9 +1505,9 @@ def _showfixtures_main(config: Config, session: Session) -> None:
         if fixturedef.scope != "function":
             tw.write(" [%s scope]" % fixturedef.scope, cyan=True)
         if verbose > 0:
-            tw.write(" -- %s" % bestrel, yellow=True)
+            tw.write(" -- %s" % str(bestrel), yellow=True)
```

Comment:
Is the `str` redudant?

Reply:
since namedtuple isinstance tuple, %-formatting will try and unpack it.  if this converts to f-string you won't need `str(...)`